docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            bures_cost: An instance of the Bures cost class.
</pre><p>I think bures_cost should not be a parameter, since you are only using a function of the class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>ok</p>